pkg/runtime/interfaces/application_configuration.go

```diff
@@ -149,6 +149,10 @@ type NotificationsConfig struct {
 	NotificationsPublisherConfig NotificationsPublisherConfig `json:"publisher"`
 	NotificationsProcessorConfig NotificationsProcessorConfig `json:"processor"`
 	NotificationsEmailerConfig   NotificationsEmailerConfig   `json:"emailer"`
+	// Number of times to attempt recreating a notifications processor client should there be any disruptions.
+	ReconnectAttempts int `json:"reconnectAttempts"`
+	// Specifies the time interval to wait before attempting to reconnect the notifications processor client.
+	ReconnectDelaySeconds int `json:"reconnectDelaySeconds"`
```

Comment:
should this be int?  We can make it this if you want https://github.com/lyft/flytestdlib/blob/537f86093d86af270aab300742ee1a56f2905885/config/duration.go#L10

Reply:
sure, but it's a question of how much granularity we need to expose - are people really going to configure retry delays on the order or many minutes or even hours?